packages/fx-core/src/question/options/AddPluginOptions.ts

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/****************************************************************************************
+ *                            NOTICE: AUTO-GENERATED                                    *
+ ****************************************************************************************
+ * This file is automatically generated by script "./src/question/generator.ts".        *
+ * Please don't manually change its contents, as any modifications will be overwritten! *
+ ***************************************************************************************/
+
+import { CLICommandOption, CLICommandArgument } from "@microsoft/teamsfx-api";
+
+export const AddPluginOptions: CLICommandOption[] = [
+  {
+    name: "teams-manifest-file",
+    questionName: "manifest-path",
+    type: "string",
+    shortName: "t",
+    description:
+      "Specifies the Microsoft Teams app manifest template file path, it can be either absolute path or relative path to project root folder, defaults to './appPackage/manifest.json'",
```

Comment:
Specify the path for Teams app manifest template. It can be either absolute path or relative path to the project root folder, with default at './appPackage/manifest.json'.